registry/quilt_server/models.py

```diff
@@ -190,6 +190,29 @@ def __str__(self): return '%d' % self
 
 db.Index('idx_package', Event.package_owner, Event.package_name)
 
+class User(db.Model):
+    id = db.Column(postgresql.UUID, primary_key=True)
+    name = db.Column(db.String(64), unique=True)
+    email = db.Column(db.String(64), nullable=False, unique=True) # should this be longer?
+    password = db.Column(db.String(200))
+    is_admin = db.Column(db.Boolean, default=False)
+    last_login = db.Column(postgresql.TIMESTAMP(True), server_default=db.func.now())
+    first_name = db.Column(db.String(64))
+    last_name = db.Column(db.String(64))
+    is_active = db.Column(db.Boolean, default=True)
+    date_joined = db.Column(postgresql.TIMESTAMP(True), server_default=db.func.now(), nullable=False)
+    old_id = db.Column(db.BigInteger) # for django ID -- probably not necessary but good to keep around
+
+class Code(db.Model):
+    # each user can have only one code, so only user_id is primary key
```

Comment:
Because otherwise you have a bunch laying around, when you should always just get a new one and revoke the old one. They're only supposed to exist for a minute tops when used normally